pkg/config/app_config.go

```diff
@@ -323,7 +323,7 @@ type AppState struct {
 	HideCommandLog             bool
 	IgnoreWhitespaceInDiffView bool
 	DiffContextSize            int
-	RemoteBranchSortOrder      string
+	RemoteBranchSortOrder      string `yaml:"remoteBranchSortOrder" jsonschema:"enum=alphabetical,enum=date"`
```

Comment:
I'd drop this change again. We don't have json schema support for state.yml, and there's little reason to add it since this is not a file that users usually edit. And as for the `yaml` tag: it's true that without this the key will be written as `remotebranchsortorder` which is quite ugly; but we are already doing this for all the other keys, and can't easily change that for backwards compatibility reasons. It's also just a cosmetic issue, really.

Reply:
That's a good point, i have no problem with lowercase name, I was trying to find a place to list valid `enum=` values where they can be visible at a glance, but this is not user oriented anyway. I'll clean it up.

Reply:
Removed in https://github.com/jesseduffield/lazygit/pull/3171/commits/ca34bacc60ccf30db8da57eb1f2a8301193788b4